### ISSUE TITLE
Add docs to verify setup/next steps

### ISSUE THREAD

**Issue by faximan** [abcxyz/github-metrics-aggregator#257]
### TL;DR

After going through the README I am not sure what to do next.

### Additional information

Terraform apply applied cleanly and the two Cloud Run services are green. After that I installed the GitHub app into my org (PS I noticed this is not explicitly mentioned as a required step).

But what do I do then? I see a lot of tables in BigQuery, but all of them are empty (e.g. "raw_events"). Should they be?

There is a section on Looker Studio in the README, but I am not quite sure how to use that query. My end goal is to get something like [http://go/bp-metrics-public](https://www.google.com/url?q=http://go/bp-metrics-public&sa=D&source=buganizer&usg=AOvVaw18156vtCQra8Nt4QY9ogU4) set up, but I don't know how to.
